yeyun/app.js

```diff
@@ -115,6 +116,32 @@ app.post("/posts", async (req, res) => {
   res.status(201).json({ message: "postCreated" });
 });
 
+app.post("/signin", async (req, res) => {
+  const { email, password } = req.body;
+
+  const [userData] = await appDataSource.query(
+    `SELECT * FROM users WHERE email = ?`,
+    [email]
+  );
+
+  if (!userData) {
+    return res.status(401).json({ message: "Inavalid User" });
+  }
+
+  const result = await bcrypt.compare(password, userData.password);
+
+  if (!result) {
+    return res.status(401).json({ message: "Invalid User" });
+  }
+
+  const jwtToken = await jwt.sign(
+    { userId: userData.id },
+    process.env.SECRETKEY
+  );
```

Comment:
jwt.sign에 인자로 payload 를 변수로 만들어 직접적으로 나타나지 않도록 수정해 주세요.
```suggestion
const payload = { userId: userData.id }
 const jwtToken = await jwt.sign(
    payload,
    process.env.SECRETKEY
  );
```